symengine/pow.cpp

```diff
@@ -94,7 +94,12 @@ RCP<const Basic> pow(const RCP<const Basic> &a, const RCP<const Basic> &b)
         return pownum(rcp_static_cast<const Number>(b), zero);
     }
     if (eq(*b, *one)) return a;
-    if (eq(*a, *zero)) return zero;
+    if (eq(*a, *zero) && is_a_Number(*b)) {
```

Comment:
But at `b = -1`, `a` will be indeterminate.
I think, I am not able to understand this. Could you please elaborate ?